styles/dropdown.module.scss

```diff
@@ -0,0 +1,12 @@
+.trigger {
+border-style: none;
+
+}
+
+.content {
+  background-color: white;
+}
+
+.item {
+
+}
```

Comment:
You can safely delete lines 10-12. We can recreate it later if we need to put a style in it.